src/fights.ts

```diff
@@ -1021,7 +1037,12 @@ const freeFightSources = [
               $item`Louder Than Bomb`,
             ),
           )
-            .if_($monster`cat-alien`, Macro.trySkill($skill`Snokebomb`).tryItem($item`tennis ball`))
+            .if_(
+              $monster`cat-alien`,
+              get("_snokebombUsed") < snokeLimit
+                ? Macro.trySkill($skill`Snokebomb`).item($item`tennis ball`)
+                : Macro.item($item`tennis ball`),
```

Comment:
This seems like a piece of logic we're using a lot; consider creating a bespoke Macro method for it. Or something.